src/pages/Diet/components/AddImage.jsx

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+export function AddImage() {
+  let [link, setLink] = useState('');
+  let [files, setFiles] = useState(null);
+
+  function handleFileChange(e) {
+    const { files } = e.target;
+    setFiles(files[0]);
+
+    if (files.length === 0) {
+      setLink('');
+    } else {
+      const imageURL = window.URL.createObjectURL(files[0]);
+      setLink(imageURL);
+    }
+  }
```

Comment:
although we know the `files` here is come from the `e.target`, but this variable name is as same as `line 6` useState's name, that will be confuse to see two same variable name.